operations-stdlib/src/commonMain/kotlin/string/Trim.kt

```diff
@@ -0,0 +1,54 @@
+package string
+
+import operation.StandardLogicOperation
+import utils.asList
+
+object Trim : StandardLogicOperation, StringUnwrapStrategy {
+    private const val ARGUMENTS_ARG_INDEX = 0
```

Comment:
if you apply changes from comments below this will become unused

Reply:
Done